include/jemalloc/internal/edata.h

```diff
@@ -193,16 +194,11 @@ struct edata_s {
 	};
 
 	/*
-	 * Reserved for hugepages -- once that allocator is more settled, we
-	 * might be able to claw some of this back.  Until then, don't get any
-	 * funny ideas about using the space we just freed up to keep some other
-	 * bit of metadata around.  That kind of thinking can be hazardous to
```

Comment:
Glad that we remove one hazard for the developers!

Reply:
Vague threats of bodily harm are an important part of the development process.